src/inspector_agent.h

```diff
@@ -23,6 +25,25 @@ namespace inspector {
 
 class AgentImpl;
 
+class ContextInfo {
+ public:
+  explicit ContextInfo(v8::Local<v8::Context> context, const int groupId,
+                       const char* name)
+                       : group_id_(groupId),
+                       name_(name) {
+                       context_.Reset(context->GetIsolate(), context);
+  }
+  inline v8::Local<v8::Context> context(v8::Isolate* isolate) const {
+    return context_.Get(isolate);
+  }
+  int groupId() const { return group_id_; }
```

Comment:
is there a style guide on this or reason why linting doesn't pick these things up?